src/yea_wandb/mock_server.py

```diff
@@ -499,6 +499,19 @@ def graphql():
                     }
                 }
             )
+        if "__type" in body["query"]:         
```

Comment:
change this to the more specific introspection query you proposed.

Reply:
ha ha... just did it :)